README.md

```diff
@@ -9,6 +9,8 @@
 
 This is a curated list of demos that showcase Apache Kafka® event stream processing on the Confluent Platform, an event stream processing platform that enables you to process, organize, and manage massive amounts of streaming data across cloud, on-prem, and serverless deployments.
 
+<a href="http://www.youtube.com/watch?feature=player_embedded&v=muQBd6gry0U" target="_blank"><img src="http://img.youtube.com/vi/muQBd6gry0U/0.jpg" width="240" height="180" border="10" /></a>
```

Comment:
maybe look into making the demo preview larger and centered?  It feels a little slapped on in the current orientation IMO.

Reply:
* I made it a little bigger.
* I had tried for centering it but didn't figure it out.  Moving on ;)